src/index.html

```diff
@@ -11,7 +11,116 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="big_margin">
+        <legend>Personal information</legend>
+        <label class="field small_margin">Surname:
+          <input
+          type="text"
+          name="surname"
+          >
+        </label>
+        <label class="field small_margin">Name:
+          <input
+          type="text"
+          name="name"
+          >
+        </label>
+        <label class="field small_margin">How old are You?
+          <input
+          type="number"
+          name="age"
+          min="1" max="100"
+          value="12"
+          >
```

Comment:
```suggestion
          <input
            type="number"
            name="age"
            min="1" max="100"
            value="12"
          >
```
don't forget about indentation (everywhere) 😉